x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/alerts/basic_license_essentials_tier/ess_specific_index_logic/migrations/finalize_alerts_migrations.ts

```diff
@@ -190,7 +190,7 @@ export default ({ getService }: FtrProviderContext): void => {
     });
 
     // This fails and should be investigated or removed if it no longer applies
-    it.skip('deletes the underlying migration task', async () => {
```

Comment:
This looked familiar: I wrote this test, and it's been skipped since [it was originally introduced](https://github.com/elastic/kibana/pull/85690). I don't recall exactly why it was skipped, and it doesn't look like I left any clues for us 😢 .

However, I was surprised to see the build was green after unskipping this. It looks to be a false positive, as the actual assertions of this test are commented out below (and most obviously, no `getMigration` helper currently exists).

I would say: just keep this skipped for now, and (maybe) create a ticket to unskip it in the future.

Reply:
Tracking https://github.com/elastic/kibana/issues/179593